pkg/ebpf/capture.go

```diff
@@ -136,7 +135,7 @@ func (t *Tracee) handleFileCaptures(ctx context.Context) {
 					t.handleError(err)
 					continue
 				}
-				bpfName := string(bytes.TrimRight(bpfObjectMeta.Name[:], "\x00"))
+				bpfName := utils.TrimTrailingNUL(bpfObjectMeta.Name[:])
```

Comment:
missing string conversion?

Reply:
Not actually, since it's finally converted below via `fmt.Sprintf`, so we avoid a discarded string conversion here.